### PR TITLE
refactor: Set up import/order eslint rule for workflow

### PR DESCRIPTION
**Pull request by tomi**
## Summary

Enable `import/order` eslint rule in the workflow package

Related PR:
https://github.com/n8n-io/n8n/pull/10847

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/workflow/.eslintrc.js [hunk 1]
```diff
@@ -11,8 +11,19 @@ module.exports = {
 	rules: {
 		complexity: 'error',
 
+		'import/order': [
```
Comment (netroy) [inline packages/workflow/.eslintrc.js:14]: can we define these rules in the base config instead?

Comment (tomi) [reply packages/workflow/.eslintrc.js:14]: Yes. That requires either fixing all `@n8n/` packages or turning the rule off for them. I'll move it the base config and fix those packages in a follow-up PR, if that's OK?

Comment (tomi) [reply packages/workflow/.eslintrc.js:14]: Here: https://github.com/n8n-io/n8n/pull/10852

**Comment by tomi** [comment]
Rebased on top of https://github.com/n8n-io/n8n/pull/10847

**Review by netroy — APPROVED** [review]

**Pull request merged**